web/src/components/language/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+export { default as LanguageSelector } from "./LanguageSelector";
```

Comment:
Maybe `LanguageSelector` can be named just `Selector` now that it is part of the _language_ module. On the other hand, then it might or might not be imported as _LanguageSelector_ or so in some places.

Reply:
I would do all those changes about renaming things as part of a separate PR.